sceptre/cli/list.py

```diff
@@ -127,3 +126,42 @@ def list_change_sets(ctx, path, url):
 
     for response in responses:
         write(response, context.output_format)
+
+
+@list_group.command(name="stacks")
+@click.option(
+    "--all", is_flag=True, help="List all stacks in stack groups."
+)
+@click.argument("path")
+@click.pass_context
+@catch_exceptions
+def list_stacks(ctx, path, all):
+    """
+    List stack names and command paths for stacks in a
+    stack group.
+    \f
+
+    :param path: Path to execute the command on.
+    :type path: str
+    :param all: List all stacks in stack groups.
+    :type all: bool
```

Comment:
Remember, for click commands, we shouldn't have type docstrings, since the docs are more appropriately rendered by Click.